requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest==6.2.5
+spaudiopy==0.1.4
+librosa==0.8.1
+soundata==0.1.0
+mock==4.0.3
```

Comment:
perhaps remove this, since it's only used for testing. Other than that, it looks very good! Thanks